examples/threading-multicore/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "threading-multicore"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+publish = false
+
+[dependencies]
+riot-rs = { path = "../../src/riot-rs", default-features = false, features = [
+  "threading",
+  "multicore",
+  "core-affinity",
+] }
+riot-rs-boards = { path = "../../src/riot-rs-boards" }
+riot-rs-rt = { path = "../../src/riot-rs-rt", features = ["_panic-handler"] }
```

Comment:
Should `_panic-handler` be a feature on `riot-rs` so we don't have to import `riot-rs-rt` here?

Reply:
I've added `single-core` to the default features of `riot-rs`, so for `multicore`we have to disable it. The alternative would be to not make `single-core` a default, but then it'd have to enabled manually every time.

Reply:
Solved now with laze instead: 78171d80b228bd4322d7f6ede5c85c1180ccf475.